start-picsure.sh

```diff
@@ -77,14 +96,31 @@ docker run --name=wildfly --restart always --network=picsure -u root \
 # causing "Device or resource busy" errors during hot deployments. We just copy the files into the running container.
 docker cp "${DOCKER_CONFIG_DIR}/wildfly/deployments/." "wildfly:/opt/jboss/wildfly/standalone/deployments/"
 
-docker stop dictionary-api && docker rm dictionary-api
-docker run --name dictionary-api --restart always \
- --network=picsure --network=dictionary --network=hpdsNet \
- --env-file $CURRENT_FS_DOCKER_CONFIG_DIR/dictionary/dictionary.env \
- -d avillach/dictionary-api:latest \
- || exit 2
+if $INCLUDE_UPLOADER; then
+  docker compose --profile production -f $CURRENT_FS_DOCKER_CONFIG_DIR/uploader/docker-compose.yml up -d
+fi
+
+if $INCLUDE_DICTIONARY; then
+  docker start dictionary-db
+  docker stop dictionary-api && docker rm dictionary-api
+  docker run --name dictionary-api --restart always \
+   --network=picsure --network=dictionary \
+   --env-file $CURRENT_FS_DOCKER_CONFIG_DIR/dictionary/dictionary.env \
+   -d avillach/dictionary-api:latest \
+   || exit 2
+fi
+
+if $INCLUDE_AGG_DICT; then
+  docker stop dictionary-dump && docker rm dictionary-dump
+  docker run --name dictionary-api --restart always \
+    --network=dictionary \
+    --env-file $CURRENT_FS_DOCKER_CONFIG_DIR/dictionary/dictionary.env \
+    -v $DOCKER_CONFIG_DIR/dictionary/dump/application.properties:/application.properties \
+    -d avillach/dictionary-dump:latest \
+   || exit 2
```

Comment:
Interesting. I didn't really know there were more than two error codes for shell. :O Always learning new things!

[TLDP: Exit Codes With Special Meanings](https://tldp.org/LDP/abs/html/exitcodes.html) or [RedHat: Bash command line exit codes demystified](https://www.redhat.com/en/blog/exit-codes-demystified)